features/integration/simulate.feature

```diff
@@ -0,0 +1,84 @@
+Feature: Simulating transactions
+  Background:
+    Given an algod v2 client
+    And a kmd client
+    And wallet information
+    And suggested transaction parameters from the algod v2 client
+    And I create a new transient account and fund it with 100000000 microalgos.
+    And I make a transaction signer for the transient account.
+
+  @simulate
+  Scenario Outline: Simulating successful payment transactions
+    Given default transaction with parameters <amt> "<note>"
+    When I get the private key
+    And I sign the transaction with the private key
```

Comment:
This is a good point. I originally omitted missing signature tests in the SDKs because some of them currently do not allow unsigned transactions to be encoded and sent over to algod at all if they are missing a signature. In the JS SDK, the encoder throws an error if there is a missing field (signature) in the transaction: https://github.com/algorand/js-algorand-sdk/blob/develop/src/encoding/encoding.ts#L46 

I still think simulating transactions without signatures sounds like a valid use-case so we might have to support this some other way. Tagging @barnjamin as well, and I'll also think about this a bit more.

Reply:
see thread here https://github.com/algorand/py-algorand-sdk/pull/420#discussion_r1122379707

maybe add a new EmptySigner

Reply:
I'll make an issue on this as well (support missing sigs in the SDKs and simulation) - we can address this in a follow-up PR?

Reply:
thx for issue #267 

resolving